task5/task5/src/main/java/classes/NameServlet.java

```diff
@@ -0,0 +1,21 @@
+package classes;
```

Comment:
To avoid posting the same files over and over again, you can create a pull request not into master but into the (previous) unmerged branch. It doesn't spoil anything. If you then merge the "previous" branch, your PR automatically switches to the master target